tests/migration/online_migration/yast2_migration.pm

```diff
@@ -152,7 +152,8 @@ sub run {
     }
 
     my $migration_cmd = get_var('LEAP_TECH_PREVIEW_REPO') ? 'migration_sle' : 'migration';
-    script_run("yast2 $migration_cmd; echo yast2-migration-done-\$? > /dev/$serialdev", 0);
+    my $yast_cmd = (is_ppc64le and is_pvm_hmc) ? 'yast' : 'yast2';
```

Comment:
why need is_ppc64le? I suppose only is_pvm_hmc is enough

Reply:
`yast` force ncurses even when qt is available. `yast2` prefers graphical one if it is available.

Reply:
made the condition simple, fixed